src/modm/driver/gpio/pca8574.hpp

```diff
@@ -73,9 +73,12 @@ class Pca8574 : public pca8574, public modm::I2cDevice< I2cMaster, 2 >, public m
 	modm::ResumableResult<bool> inline
 	setOutput(Pins pins)
 	{
-		// nothing needs to be changed for switching to output
 		direction.set(pins);
-		return {modm::rf::Stopped, true};
+#ifdef MODM_RESUMABLE_IS_FIBER
+		return true;
+#else
+		return {modm::rf::Stop, true};
+#endif
```

Comment:
Why not use the same solution as in [`src/modm/driver/gpio/mcp23_transport_impl.hpp`](https://github.com/modm-io/modm/pull/1001/files?diff=split&w=0#diff-ae278c5a317372b744a3cc500dbfde7b45d8b8dd42a22dada5d8f2f5f817f679)?
```suggestion
	RF_BEGIN();
	direction.set(pins);
	RF_RETURN(true);
```

Reply:
Because the driver does not inherit from a Resumable class and the Macros set the internal state, so it doesn't compile. 😢

Reply:
oh, my bad, it actually does inherit via I2cDevice!